functions/users/index.js

```diff
@@ -0,0 +1,102 @@
+const functions = require('firebase-functions');
+const admin = require('firebase-admin');
+const dbref = admin.firebase().ref();
+
+
+function checkUserData(data){
+    return data.hasOwnProperty('first_name')
+        && data.hasOwnProperty('last_name')
+        && data.hasOwnProperty('email')
+        && data.hasOwnProperty('password')
+        && data.hasOwnProperty('phone')
+        && data.hasOwnProperty('address')
+        && data.hasOwnProperty('postal_code')
+        && data.hasOwnProperty('country')
+        && data.hasOwnProperty('city')
+        && data.hasOwnProperty('date_of_birth')
+        && data.hasOwnProperty('gender');
+}
+
+exports.createUser = functions.https.onRequest((req, res) => {
+    if(req.method !== 'POST') {
+        return res.status(400).json({
+            message: 'Bad request, POST required'
+        })
+    }
+    else {
+        const data = req.body;
+        if (!checkUserData(data)) {
+            return res.status(400).json({
+                message: 'Missing parameters ' +
+                    'required: ' +
+                    'first_name, ' +
+                    'last_name, ' +
+                    'email, ' +
+                    'password, ' +
+                    'phone, ' +
+                    'address, ' +
+                    'postal_code, ' +
+                    'city, ' +
+                    'country, ' +
+                    'date_of_birth, ' +
+                    'gender'
+            })
+        }
+        admin.auth().createUser({
+            email: data.email,
+            emailVerified: false,
+            password: data.password,
+            disabled: false
+        })
+            .then((userRecord) => {
+                console.log("User created with UID of: " + userRecord.uid);
```

Comment:
Prefer template strings to concatenation, 
```suggestion
                console.log(`User created with UID of: ${userRecord.uid}`);
```